provider/aws/init.go

```diff
@@ -5,20 +5,33 @@ import (
 	"github.com/aws/aws-sdk-go/aws/credentials/stscreds"
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/stelligent/mu/common"
+	"net/http"
+	"net/url"
 	"os"
 )
 
 // InitializeContext loads manager objects
-func InitializeContext(ctx *common.Context, profile string, assumeRole string, region string, dryrunPath string, skipVersionCheck bool) error {
+func InitializeContext(ctx *common.Context, profile string, assumeRole string, region string, dryrunPath string, skipVersionCheck bool, proxy string) error {
 
 	sessOptions := session.Options{SharedConfigState: session.SharedConfigEnable}
+
 	if region != common.Empty {
 		sessOptions.Config = aws.Config{Region: aws.String(region)}
 	}
+	if proxy != common.Empty {
+		proxyHTTPClient := &http.Client{
+			Transport: &http.Transport{
+				Proxy: func(*http.Request) (*url.URL, error) {
+					return url.Parse(proxy)
+				},
+			},
+		}
+		sessOptions.Config = aws.Config{HTTPClient: proxyHTTPClient}
```

Comment:
line 29 will conflict with line 19.  If you pass both `--region` and `--proxy` then the `--region` will be ignored.